### DISCUSSION TITLE
idk whats wrong lol

### DISCUSSION THREAD

**Discussion by PupperwareParty** [nickcoutsos/keymap-editor#189] [Q&A] [answered]
![image](https://github.com/nickcoutsos/keymap-editor/assets/161689692/2070025e-b78c-4df0-a4cd-fac3ee12e3b4)

not sure what im doing wrong pls help. just trying to make a lily58

**Comment by PupperwareParty** [comment]
![image](https://github.com/nickcoutsos/keymap-editor/assets/161689692/f3112874-ac49-40b4-8889-8e6bab63b6e8)
from the builder

**Reply by nickcoutsos** [reply, marked as answer]
It doesn't really look like anything is obviously wrong to me. The build was successful, and the firmware can be downloaded [here](https://github.com/PupperwareParty/lily58-wireless-view-zmk-config/actions/runs/8092575431) if that's your concern. If not I'll need a little more to go on in order to offer any help.

**Reply by nickcoutsos** [reply]
@PupperwareParty any luck?

**Comment by PupperwareParty** [comment]
It looked like it just froze, i re submitted and it worked, thank you so
much for your help!
